src/styles/searchResults.css

```diff
@@ -0,0 +1,18 @@
+.space-image{
+    width: 25%;
+  margin: 30px;
+  padding: 15px;
+  border-radius: 25px;
+}
```

Comment:
Indentation is a bit off, also the opening bracket should have a space before it (this isthe same for all classes in this file)

```
.space-image {
    width: 25%;
    margin: 30px;
    padding: 15px;
    border-radius: 25px;
}
```